account/wallet_test.go

```diff
@@ -0,0 +1,25 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"testing"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateAddress(t *testing.T) {
+	addr, pub, priv, err := CreateAddress()
+	require.NoError(t, err)
+	require.Equal(t, gethcommon.IsHexAddress(addr), true)
+
+	privECDSA, err := crypto.HexToECDSA(priv[2:])
```

Comment:
The problem is that `hexutil` is used in `CreateAddress` while `crypto` uses `hex` from standard lib which does not expect `0x` prefix. 

Now, the question is if `CreateAddress` should return nicely formatted strings or instead it would return a private key and the whole formatting for JSON-RPC API response would happen in `func (api *PublicAPI) CreateAddress(context context.Context)`. I think that the second option is nicer.

Also, is the JSON-RPC method expected to return byte arrays encoded with `hexutils` and contain `0x` prefix?

Reply:
The JSON-RPC method is not specified. I'm implementing a new method to address https://github.com/status-im/status-react/issues/6875.

Reply:
RIght but I am talking about following a convention which is used in other existing methods. 

But yeah, `hexutils` seems like a way to go, for example: `"result": "0xc94" // 1207`

Reply:
Let's merge as it is and we can have a follow-up.